trin-history/src/lib.rs

```diff
@@ -113,3 +116,16 @@ pub fn spawn_history_network(
             .expect("failed to pause until ctrl-c");
     })
 }
+
+pub fn spawn_history_heartbeat(network: Arc<HistoryNetwork>) {
+    tokio::spawn(async move {
+        let mut heart_interval = interval(Duration::from_millis(30000));
+
+        loop {
+            let radius = network.overlay.data_radius();
```

Comment:
Pretty sure this is wrong, and it needs to be:
```suggestion
            let radius = network.overlay.store.read().data_radius();
```

It seems that the radius on the overlay is just the user-supplied thing, and never changes.

Reply:
There we go, there's a rounding bug right now, which I'd like to fix before merging.

Reply:
Maybe the `network.overlay.data_radius()` should be renamed to `hardcoded_data_radius()` or something.

Reply:
Yup! Tracking that thought (or removing it entirely) in https://github.com/orgs/ethereum/projects/33/views/1?pane=issue&itemId=23912052